src/components/Register/index.tsx

```diff
@@ -1,56 +1,166 @@
-            <input className="block w-full h-12 pl-8 border-b-[2px] border-[#9caccb] placeholder:text-[#9caccb]" placeholder="Insira seu username" type="text" name="username" id="username"/>
-          </div>
 
-          <label className="block mt-6" htmlFor="password">Senha</label>
-          <div className="relative">
-            <svg className="absolute top-2.5" xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="#9caccb" viewBox="0 0 256 256"><path d="M208,82H174V56a46,46,0,0,0-92,0V82H48A14,14,0,0,0,34,96V208a14,14,0,0,0,14,14H208a14,14,0,0,0,14-14V96A14,14,0,0,0,208,82ZM94,56a34,34,0,0,1,68,0V82H94ZM210,208a2,2,0,0,1-2,2H48a2,2,0,0,1-2-2V96a2,2,0,0,1,2-2H208a2,2,0,0,1,2,2Zm-82-94a26,26,0,0,0-6,51.29V184a6,6,0,0,0,12,0V165.29A26,26,0,0,0,128,114Zm0,40a14,14,0,1,1,14-14A14,14,0,0,1,128,154Z"></path></svg>
-            <input className="block w-full h-12 pl-8 border-b-[2px] border-[#9caccb] placeholder:text-[#9caccb]" placeholder="Insira sua senha" type="password" name="password" id="password"/>
+  return <>
+    <Container>
+      <div className="h-screen  flex justify-center items-center">
+        <main className="w-[875px] flex divide-x-2 gap-6  p-16 placeholder:text-[#9caccb] bg-white shadow-lg rounded-md">
+          <div>
+            <h1 className="mb-12 text-center text-black font-bold ">Sign Up</h1>
+
+            <form onSubmit={handleSignUp}>
+
+              <label className="block mt-6 text-black" htmlFor="username">Nome</label>
+              <div className="relative w-full">
+                <input className=" text-slate-500 block w-full h-12 border-b-[2px] border-[#9caccb] " placeholder="Insira seu username" type="text" name="username" id="username" value={username} onChange={handleUsernameChange} />
+              </div>
+
+              <label className="block mt-6 text-black" htmlFor="password">Senha</label>
+              <div className="relative flex gap-4 w-full">
+                {
+                  isVisibility ?
+                    <><input className="block text-slate-500 w-full h-12 border-b-[2px] border-[#9caccb] " placeholder="Insira sua senha" type="password" name="password" id="password" required={true} value={password} onChange={handlePasswordChange} /><Image src={visibilityOn} alt="icone para visualizar a senha" className="size-10" onClick={() => setIsVisibility(!isVisibility)} /></>
+                    :
+                    <><input className="block text-slate-500 w-full h-12 border-b-[2px] border-[#9caccb] " placeholder="Insira sua senha" type="text" name="password" id="password" required={true} value={password} onChange={handlePasswordChange} />
+                      <Image src={visibilityOff} alt="icone para visualizar a senha" className="size-10" onClick={() => setIsVisibility(!isVisibility)} /></>
+                }
+                {/* <SafeImage src={visibilityOn} text={"icone para visualizar a senha"}  /> */}
```

Comment:
se não for mais utilizar é preferível remover de vez essa linha mesmo, evitar deixar como comentário